server/apps/distributed-app/docs/modules/ROOT/pages/configure/imap.adoc

```diff
@@ -44,7 +44,12 @@ Defaults to 10MB. Must be a positive integer, optionally with a unit: B, K, M, G
 Defaults to 0 (unlimited). Must be a positive integer, optionally with a unit: B, K, M, G.
 
 | plainAuthDisallowed
-| Whether to enable Authentication PLAIN if the connection is not encrypted via SSL or STARTTLS. Defaults to `true`.
+| Deprecated. Should use `auth.plainAuthEnabled`, `auth.requireSSL` instead.
```

Comment:
Can we remove `auth.plainAuthEnabled` part? IMO only `auth.requireSSL` is corresponding enough to plainAuthDisallowed.

Reply:
No IMO if we support a fallback it should be documented. Otherwise people might get confused...